src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -106,7 +106,7 @@ private Object readResolve() {
                     merge,
                     metadata.getNumber(),
                     new BranchSCMHead(metadata.getBaseRef()),
-                    metadata.getUserLogin(),
+                    null,
```

Comment:
Well if you try to use the SCMFilesystem api on a PR before the index, it may say it is not supported...

But as you have to handle the case anyway (e.g. there are certain classes of PR which we cannot support with the SCMFilesystem api) should be not an issue